app/controllers/request_controller.rb

```diff
@@ -28,6 +28,24 @@ def new
 
     @holding, @item = cat_services_client.get_holding(instance_id: @instance_id, holdings_id: @holdings_id, item_id: @item_id)
 
+    format = @document.fetch("format")
+    @request_form =
+      if format.present? && @item["enumeration"].present? && @item["enumeration"].include?("USE THE REQUEST OPTION BELOW")
+        if format.first == "Manuscript"
+          "manuscripts_serial"
+        else
+          "journals"
+        end
+      else
+        "monograph"
+      end
+
```

Comment:
I don't know if this of help, but at the item level in the instance hierarchy JSON, there is an itemCategory property which flags the type.